src/components/ReportActionItem/MoneyRequestPreview/MoneyRequestPreviewContent.tsx

```diff
@@ -294,8 +294,10 @@ function MoneyRequestPreviewContent({
 
     const navigateToReviewFields = () => {
         const backTo = route.params.backTo;
-        const comparisonResult = TransactionUtils.compareDuplicateTransactionFields(reviewingTransactionID);
-        Transaction.setReviewDuplicatesKey({...comparisonResult.keep, duplicates, transactionID: transaction?.transactionID ?? ''});
+        Transaction.abandonReviewDuplicateTransactions();
```

Comment:
I approved, but now that I think about it, can you add a comment before this line explaining why we need to call `Transaction.abandonReviewDuplicateTransactions();`?